sample.tf

```diff
@@ -157,6 +157,23 @@ resource "artifactory_local_sbt_repository" "sbt-local" {
   description = "Repo created by Terraform Provider Artifactory"
 }
 
+resource "artifactory_local_terraform_module_repository" "terraform-module-local" {
+  key           = "terraform-module-local"
+  description   = "Repo created by Terraform Provider Artifactory"
+  registry_type = "module"
+}
+
+resource "artifactory_local_terraform_provider_repository" "terraform-provider-local" {
+  key           = "terraform-provider-local"
+  description   = "Repo created by Terraform Provider Artifactory"
+  registry_type = "provider"
```

Comment:
This is redundant - you already have `provider` in the resource type

Reply:
removed